src/routes/ticket-info.svelte

```diff
@@ -164,15 +164,7 @@
     Groups of 25 or more may purchase tickets at the rate noted above. Tickets
     may be added to a group sale (if seats are available) up to show date but
     are never reduced or refunded. The Post Playhouse cannot be responsible for
-    unused group tickets. Group rate tickets can not be purchased online. Please
-    email us for group rate ticket needs at
-    <a class="link-green" href="mailto:tickets@postplayhouse.com">
-      tickets@postplayhouse.com
-    </a>
-    or call our box office at
-    <a class="x" href="tel:+{site.boxOfficePhone.replace(/-/g, '')}"
-      >{site.boxOfficePhone}</a
-    >.
```

Comment:
Since this is a change from years past, let's not just remove the options. Instead, let's add that they can be purchased online.

Something like "Group tickets may be purchased the same ways you can purchase individual tickets ...[explanation of all methods]"

Reply:
Added the info back in. Ready for another review.